ui/main_menu.py

```diff
@@ -141,7 +149,7 @@ def place_images(self):
         image_uni_klinikum = PIL.Image.open(
             "ui/images/logos/UniklinikumTübingen.png")
```

Comment:
I think this image needs to be replaced so that it follows the rules of the corporate design script. Especially at the bottom there seems to be too less space. It would be great if you could fix that as well, I don't know whether you can download the "perfect" logo from somewhere. Keep in mind to adjust the image sizes at the top bar so that it looks like before (same height).